cobra/core/model.py

```diff
@@ -316,6 +316,8 @@ def copy(self):
         except Exception:  # pragma: no cover
             new._solver = copy(self.solver)  # pragma: no cover
 
+        new._contexts = list()
```

Comment:
Would be nice to have a comment to that effect so that we remember why this line is here.